db/migrate/20220912100910_create_movies.rb

```diff
@@ -0,0 +1,11 @@
+class CreateMovies < ActiveRecord::Migration[7.0]
+  def change
+    create_table :movies do |t|
+      t.string :name
```

Comment:
Do you want to have movie without name etc? I think it should have `null: false` on it :)